src/main/java/com/quantumsoft/qupathcloud/dao/spec/DatasetsPathBuilder.java

```diff
@@ -15,21 +15,35 @@
 
 package com.quantumsoft.qupathcloud.dao.spec;
 
+import static com.quantumsoft.qupathcloud.dao.Constants.DATASETS;
+import static com.quantumsoft.qupathcloud.dao.Constants.LOCATIONS;
+import static com.quantumsoft.qupathcloud.dao.Constants.PROJECTS;
+import static com.quantumsoft.qupathcloud.dao.Constants.STAGE;
+
 import com.quantumsoft.qupathcloud.exception.QuPathCloudException;
 
-import static com.quantumsoft.qupathcloud.dao.Constants.*;
+/**
+ * DatasetsPathBuilder to build a path for access to Datasets in a chosen Project and Location.
```

Comment:
"DatasetsPathBuilder builds a path to access Datasets in the specified Project and Location." ?